### ISSUE TITLE
Add servicesPath arg to init()

### ISSUE THREAD

**Issue by chapmandu** [jcberquist/aws-cfml#67]
When an application already has a directory or mapping named `services`, the constructor throws an cfc not found exception.

A `servicesPath` argument would resolve this as below.

```
public struct function init(
	string awsKey = '',
	string awsSecretKey = '',
	string defaultRegion = '',
	struct constructorArgs = {},
	struct httpProxy = {server: '', port: 80},
	string servicesPath = 'services'
) {
	this.api = new com.api(awsKey, awsSecretKey, defaultRegion, httpProxy);

	for (var service in variables.services) {
		if (StructKeyExists(arguments.constructorArgs, service)) {
			StructAppend(variables.constructorArgs[service], arguments.constructorArgs[service]);
		}
		this[service] = new '#arguments.servicesPath#.#service#'(this.api, variables.constructorArgs[service]);
	}

	return this;
}
```

In my case, I'd instantiate it like so:
```
awscfml = new services.awscfml.aws(
	awsKey = "foo",
	awsSecretKey = "bar",
	defaultRegion = "ap-southeast-2",
	servicesPath = "app.services.awscfml.services"
);
```

**Comment by jcberquist** [comment]
Interesting, is this ACF or Lucee? I take it a mapping is preferred to a relative path when resolving a component path.

**Comment by chapmandu** [comment]
I'm using Lucee. Yes I usually prefer mappings for component paths. Perhaps the argument is not named appropriately.

**Comment by jcberquist** [comment]
@chapmandu I have added a commit to support an argument named `libraryMapping` - you can use it in your situation like so:
```
awscfml = new services.awscfml.aws(
	awsKey = "foo",
	awsSecretKey = "bar",
	defaultRegion = "ap-southeast-2",
	libraryMapping = "app.services.awscfml"
);
```
This is published as `v1.25.0` - please let me know how it goes.
